### ISSUE TITLE
No request send when speciying Cookies and Headers at the same time 

### ISSUE THREAD

**Issue by NNamiDC** [ffuf/ffuf#553]
When adding cookies (both via `-b` or `-H`) together with any other header(s) (via -H) nothing (no request are send) happens.

When only adding the cookies or only the other header(s) it works as expected.

Nothing is written in debug-log and no request-response files are generated when specifying -od
 
`ffuf -w wordlist.txt -u https://REDACTED/FUZZ -b "auth-id=XXXXXXXX"`

***OK***

`ffuf -w wordlist.txt -u https://REDACTED/FUZZ -H "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) Gecko/20100101 Firefox/99.0"`

***OK***

`ffuf -w wordlist.txt -u https://REDACTED/FUZZ -H "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) Gecko/20100101 Firefox/99.0" -H "Referer: https://REDACTED/"`

***OK***

`ffuf -w wordlist.txt -u https://REDACTED/FUZZ -b "auth-id=XXXXXXXX" -H "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) Gecko/20100101 Firefox/99.0"`

***NO REQUESTS/OUTPUT***

`ffuf -w wordlist.txt -u https://REDACTED/FUZZ -H "Cookie: auth-id=XXXXXXXX" -H "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) Gecko/20100101 Firefox/99.0"`

***NO REQUESTS/OUTPUT***

Tested it on Debian 11(WSL) with FFUF 1.4.1 and Kali 2022.2 with FFUF 1.5.0

Please feel free to request more info if needed.

Best regards,

**Comment by joohoi** [comment]
I tested this out on both ffuf 1.5.0 and 1.4.1 and I cannot reproduce the issue. I even copypasted your params part in verbatim and the requests still fly. You can try adding `-mc all` to output every response regardless of the status code (the defaults drop some statuses out).

**Comment by NNamiDC** [comment]
Many thanks for your response and my excuses I did not test removing the status filtering.

The two words used in the test word list returned a status 200 via the web browser so I expected the same. Without the Cookie I received a 302 to the privacy wall and without the User Agent I received a 403 from the WAF. So all included in the filter so I did not think about this.

Today I'm no longer able to test the exact same scenario as apparently my testing was noticed and it now always return 403  also when both the cookie and user agent are specified.

I tested with several other sites and it always work as expected so probably the reason was indeed a response status filtered out. I'm very sorry I used your time for something I should have found myself.

I was confused that no requests were created although specifying the -od option but the help clearly state ***"store matched results"***. So my mistake not reading the doc carefully enough. :-)

Maybe adding the possibility to generate all requests/response could be nice, but actually that already exist by removing the filter.

Many thanks for your help and especially for the very nice tool.

Best regards,
